map.go

```diff
@@ -2,11 +2,32 @@ package lo
 
 // Keys creates an array of the map keys.
 // Play: https://go.dev/play/p/Uu11fHASqrU
-func Keys[K comparable, V any](in map[K]V) []K {
-	result := make([]K, 0, len(in))
+func Keys[K comparable, V any](in ...map[K]V) []K {
+	result := make([]K, 0)
 
-	for k := range in {
-		result = append(result, k)
+	for i := range in {
+		for k := range in[i] {
+			result = append(result, k)
+		}
+	}
+
+	return result
+}
+
+// UniqKeys creates an array of unique keys in the map.
+// Play: https://go.dev/play/p/TPKAb6ILdHk
+func UniqKeys[K comparable, V any](in ...map[K]V) []K {
```

Comment:
most likely more effective

```go
  seen := make(map[K]struct{})
	
	for i := range in {
		for k := range in[i] {
			seen[k] = struct{}{}
		}
	}
	
	result := make([]K, 0, len(seen))
	
	for k := range seen {
	    result = append(result, k)
	}
```

Reply:
similarly `UniqValues`

Reply:
with this version, the order is not guarantee

IMO, we would need a new helper (UniqKeysUnordered) ?